packages/angular/src/app/app.component.ts

```diff
@@ -11,7 +12,9 @@ export class AppComponent implements OnDestroy {
     return Object.keys(this.screen.sizes).filter((cl) => this.screen.sizes[cl]).join(' ');
   }
 
-  constructor(private authService: AuthService, private screen: ScreenService, public appInfo: AppInfoService) { }
+  constructor(private authService: AuthService, private screen: ScreenService, public appInfo: AppInfoService) {
+    setAppTheme();
```

Comment:
Why do you set theme here? Could we set theme in theme-switcher.component on ngOnInit?

Reply:
it was my suggestion to set a theme when application starts since it's a global setting (?)

Reply:
for setting theme on singleCard pages (login, set password,)